style.css

```diff
@@ -0,0 +1,574 @@
+  padding-bottom: 12px;
+  margin-top: 0;
+  margin-bottom: 0;
+  color: aliceblue;
+}
+
+.portfolio p {
+  width: 327px;
+  height: 123px;
+  padding-left: 24px;
+  color: #ffff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.icons {
+  width: 200px;
+  height: 40px;
+  padding: 24px;
+  padding-top: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+main {
+  background: white;
+}
+
+.title {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', serif;
+  font-size: 40px;
+  font-weight: 400;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 120px;
+  margin-bottom: 0;
+}
+
+.vector {
+  padding-left: 24px;
+  padding-right: 24px;
+  align-items: start;
+}
+
+.placeholder {
+  padding-top: 24px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.work {
+  padding-left: 24px;
+  margin-bottom: 0;
+  height: 36px;
+  width: 327px;
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  text-align: left;
+}
+
+.paragraph {
+  height: 147px;
+  width: 327px;
+  left: 0;
+  top: 48px;
+  padding-left: 24px;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.teclast {
+  display: flex;
+  width: 314px;
+  height: 40px;
+  gap: 12px;
+  padding-left: 24px;
+}
+
+.css-button {
+  background: white;
+  width: 31px;
+  height: 20px;
+  padding: 10px, 12px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.html-button {
+  background: white;
+  width: 67px;
+  height: 20px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.boot-button {
+  background: white;
+  width: 95px;
+  height: 20px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.ruby-button {
+  background: white;
+  width: 31px;
+  height: 20px;
+  padding: 10px, 12px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+}
+
+.orange-button {
+  display: flex;
+  width: 124px;
+  height: 48px;
+  padding: 24px;
+}
+
+.see-project {
+  height: 20px;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  background: #ff6b00;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 24px;
+  align-items: center;
+  color: #ffff;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-rows: 386px 386px 386px 386px 386px 386px;
+  grid-template-columns: 327px 327px 327px 327px v 327px 327px;
+  gap: 24px;
+}
+
+.grid-item {
```

Comment:
- [ ] Please 🙏🏻 use the `flexbox` display property to align your child's items properly it will also help you align the things with equal gaps and without adding a property to individual child items in order to maintain the layout of your project card.